framework/src/headers/ip/v6/mod.rs

```diff
@@ -353,6 +353,21 @@ impl HeaderUpdates for Ipv6Header {
     }
 }
 
+trait OgRouterAddr {
```

Comment:
same as below; we don't want to be specific here in the NB repo for og stuff. Maybe is_addr (better named?) can take in a subnet address, so it's more generic.

Reply:
i'll rename it and make it more generic. this gets tricky b/c the OG_ID is dynamic `fdaa:0:<OG_ID>:a:0::/64`

Reply:
@zeeshanlakhani i'll rename it and make it more generic. this gets tricky b/c the OG_ID is dynamic `fdaa:0:<OG_ID>:a:0::/64` CIDR is `fdaa:0:1:a:0:0:0:0-fdaa:0:1:a:ffff:ffff:ffff:ffff`. should we make the CIDR something like `fdaa:0::a:0::\96? that seems a little too permissive to me. basically anything from `fdaa:0:*` would be parsed

Reply:
Maybe it takes in two params, the longest prefix address to match and and the cidr prefix (slash number)?

Reply:
since we dont want to put anything og related into netbricks code, maybe this logic should be put in og?

Reply:
That works too, but we may be able to make this generic w/ input types?

Reply:
@drunkirishcoder yeah, my thought was making it configurable and not hardcoded as it is now. Yeah, so just the prefix segments (whatever is passed in) may be good enough for this case.